Snake/BusinessLayer/score.cs

```diff
@@ -1,44 +1,73 @@
 ﻿using DatabaseLayer;
+using System.Collections.Generic;
+using System;
 
 namespace BusinessLayer
 {
     public class Score
     {
         private static int currentScore;
         private const int NEWSCORE = 0;
-        static Database db = new Database();
+        static Database db;
+        string mazeLevel;
 
-        public Score()
+        public Score(string mazeLevel)

```

Comment:
I would probably take the enum value in here instead of a string - the ToString() could be done in here.

(I'm guessing that it is a string because of the DBTest option?)